### ISSUE TITLE
Callous Unbiased Uintagroundsquirrel

### ISSUE THREAD

**Issue by ValentineRain** [ScoreShots/templatemigration#419]
![](https://scoreshotstemplates.s3.amazonaws.com/589.jpg)

# 

* Legacy name: 
* Legacy File: https://scoreshotstemplates.s3.amazonaws.com/589.json
* Legacy Category: 
* (If Client Requested) Who? 
* (If Client Requested) Still Client? 

* Potentially Problematic Elements: 

* Long-term potential merge candidates 

### Template Types Recommendation:
* Internal
* Post-Game
* Player Highlight
* Game Preview
* Mid-Game
* Social Templates
* Opening Lineup
* Player Quote
* Stats Highlight
* Poll Showcase
* Schedule Showcase
* Bracket Showcase
* Player Signing

### Template Categories Recommendation:
* Baseball
* Basketball
* Field Hockey
* Football
* Golf
* Ice Hockey
* Lacrosse
* Motorsports
* Soccer
* Softball
* Swimming and Diving
* Tennis
* Volleyball
* Wrestling
* X-Country

**Comment by adavila17** [comment]
I would say this is more media relations/conference use **but** I can see where a school could use it for multiple sports scores.

**Comment by ValentineRain** [comment]
You're correct, however, please pay special attention to the new category list. There currently is not a media relations or conference category.

The latter is an add I've been considering but as designed, there's no distinction.

**Comment by ValentineRain** [comment]
Missing dynamic behaviors.
